cargo/cargo_build_script.bzl

```diff
@@ -79,8 +69,9 @@ def _build_script_impl(ctx):
     env = dict(ctx.configuration.default_shell_env)
 
     env.update({
```

Comment:
I notice `rustc.bzl` has `_get_rustc_env()`. Can we standardize on that instead of maintaining this separate code for setting environment variables? Or does rustc have different requirements than build scripts in terms of the environment variables that should be set?

Reply:
I've filed https://github.com/bazelbuild/rules_rust/issues/661 with all of the information I know here - let's tackle it as a follow-up :)

Reply:
SG, thanks!